javascript-light-bulb/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Lightbulb</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="light-mode">
+    <div class="container">
+      <div class="row">
+        <div class="circle-light"></div>
+        <script src="main.js"></script>
```

Comment:
This should be at the end of the `body`. It should not be inside any HTML elements